docker-compose.yml

```diff
@@ -1348,3 +1352,17 @@ services:
   #       alp:
   #     restart: ${DOCKER__RESTART_POLICY:-unless-stopped}
   #     profiles: [fhir]
+  mlflow:
+    image: ${DOCKER_IMAGE_PREFIX:-ghcr.io/ohdsi/}d2e/flow-base:${DOCKER_TAG_NAME:-develop}
+    container_name: &c31 ${PROJECT_NAME:-d2e}-mlflow
+    platform: linux/amd64
+    logging:
+      options:
+        tag: *c31
+    networks:
+      alp:
+      data:
+    ports:
```

Comment:
instead of uncommenting,  move the port exposure to` docker-compose-local `instead